README.md

```diff
@@ -92,3 +92,22 @@ Remove endpoint at runtime by providing its **name**. For example, in order to r
 ```
 
 **IMPORTANT:** Resulting changes are saved in local `.env` file for reuse.
+
+## Wallet integration
+
+Any client can connet RPC Reverse Proxy. 
+A web client, scripting tools, backend servers etc. 
+It is the same as with other Ethereum RPC providers: you need to use a user's access URL containing API key.
+The proxy support CORS what enables usage within web browsers.
+
+Users may wish to integrate with wallets, e.g. Metamask.
+Below is the example of proper configuration.
+It is convenient to create a duplicate of Mainnet network configuration.
+The only data that is specific is RPC URL that is access URL and points the proxy.
+See the example below.
+
+![Metamask](./admin/docs/metamask-1.png)
+
+In case of problems check, the best place to start the investigation is to check the network traffic,
```

Comment:
```suggestion
In case of any problems with the verification of the newly added network, the best place to start the investigation is to check the network traffic,
```